TUFTOOL_EXAMPLES.md

```diff
@@ -0,0 +1,156 @@
+# Tuftool Examples
+
+## Creating and Adding New Roles
+
+The following example creates 2 roles, role1 and role2. role1 is a delegated targets of targets and role 2 is a delegated targets of role1.
```

Comment:
Perhaps it might be simpler to say:

"`role1` is delegated from `Targets` and `role2` is delegated from `role1`"